Sources/Swift/Tools/LoadValidator.swift

```diff
@@ -0,0 +1,81 @@
+@_implementationOnly import _SentryPrivate
+import Darwin
+import Foundation
+import MachO
+
+@objc
+@_spi(Private) public final class LoadValidator: NSObject {
+    // Any class should be fine, ObjC classes are better
+    static let targetClassName = NSStringFromClass(SentryDependencyContainerSwiftHelper.self)
+    
+    // This function is used to check for duplicated SDKs in the binary.
+    // Since `SentryBinaryImageInfo` is not public and only available through the Hybrid SDK, we use the expanded parameters.
+    @objc
+    @_spi(Private) public class func checkForDuplicatedSDK(imageName: String, imageAddress: NSNumber, imageSize: NSNumber) {
+        internalCheckForDuplicatedSDK(imageName, imageAddress.uint64Value, imageSize.uint64Value,
+                                      objcRuntimeWrapper: SentryDefaultObjCRuntimeWrapper.sharedInstance(),
+                                      dispatchQueueWrapper: SentryDispatchQueueWrapper())
```

Comment:
`m`: I would prefer not having a default parameter for the `SentryDispatchQueueWrapper`, because we now don't pass it in from the `SentryBinaryImageCache`, and we create an extra instance. Instead, we should reuse the one from the `SentryDependencyContainer`. We could pass the `SentryDispatchQueueWrapper` in via a init method and do the same for the objcRuntimeWrapper. Then the `LoadValidator` doesn't have to know how to create these dependencies and the code clearly communicates it's dependencies.